src/layouts/components/Editor/components/ImageBubbleMenu/ResizeImagePopover.tsx

```diff
@@ -56,39 +56,35 @@ export const ResizeImagePopover = ({
       return
     }
 
-    const isomerImageAttrs = {
-      src: node.attrs.src,
-      alt: node.attrs.alt,
-      href: node.attrs.href,
+    const style = {
+      width: data.value,
     }
 
-    editor
-      .chain()
-      .focus()
-      .setImage({
-        width: `${data.value}%`,
-        ...isomerImageAttrs,
-      })
-      .run()
+    editor.chain().focus().setImageStyle(style).run()
 
     onClose()
   }
 
   useEffect(() => {
     if (isOpen) {
       const initialValue: string | number =
-        editor.state.selection.content().content.firstChild?.attrs.width ?? 100
+        editor.state.selection
+          .content()
+          // regex to parse the width value from style tag
+          .content.firstChild?.attrs.style?.match(/width: (\d+)%/)?.[1] ?? 100
```

Comment:
sanity check - why are taking the first element here?

Reply:
My understanding is at a given time, only one element can be selected by the user (at least that's the behaviour for our RTE currently). So the first `child` of the selected `content` is the one that user is trying to modify.